service-api/module/Application/test/Controller/IdentityControllerTest.php

```diff
@@ -124,8 +124,10 @@ public function testPassportNumber(string $passportNumber, int $status): void
     public static function passportData(): array
     {
         return [
-            ['123456789', Response::STATUS_CODE_200],
-            ['123456', Response::STATUS_CODE_400]
+            [123456788, Response::STATUS_CODE_400],
+            [123456789, Response::STATUS_CODE_400],
```

Comment:
It would be good to expand this to ensure it correctly returns NO_MATCH vs NOT_ENOUGH_DETAILS